### PR TITLE
feat: add mfa state to the administration main page

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
스크린샷 2023-11-27 10 35 28

### Things to Talk About
Plz check by commits

### CODE REVIEW

File: apps/web/src/services/administration/iam/user/lib/config.ts [hunk 1]
```diff
@@ -83,6 +83,17 @@ export const userStateColor = Object.freeze({
...
+    ON: {
+        iconColor: styles.safe,
+        textColor: styles.gray[900],
+    },
+    OFF: {
+        iconColor: styles.gray[400],
+        textColor: styles.gray[400],
+    },
+});
+
```
Comment (yuda110) [inline apps/web/src/services/administration/iam/user/lib/config.ts:96]: nitpick:
```
export const userMfaColor = {
    ON: {
        iconColor: styles.safe,
        textColor: styles.gray[900],
    },
    OFF: {
        iconColor: styles.gray[400],
        textColor: styles.gray[400],
    },
} as const;
```

Comment (sulmoJ) [inline apps/web/src/services/administration/iam/user/lib/config.ts:96]: convention: we use 'as const' instead of `Object.freeze`!

**Review by sulmoJ — APPROVED** [review]
LGTM~
I left a convention comment.

**Review by yuda110 — APPROVED** [review]
LGTM! I left comments! 👍

**Pull request merged**